dynamic_rest/fields/content_type.py

```diff
@@ -0,0 +1,91 @@
+import traceback
```

Comment:
Maybe call this `dynamic_rest.fields.generic` for consistency with the field name?